flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+    "skip-icons-check": true,
+    "automerge-flathubbot-prs": true
```

Comment:
this is not needed since the is no `x-data-checker` setup

Reply:
Removed :)